components/dashboard/src/Login.tsx

```diff
@@ -126,6 +136,12 @@ export function Login() {
                         </div>
                     </div>
                 </div>
+                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
+                    {`Email address is already in use. Please log in with ${emailTaken.otherUser.authHost}.`}
```

Comment:
**suggestion:** What do you think of not surfacing this kind of information to anonymous users? Feel free to rephrase.

```suggestion
                    {`Email address already exists. Log in using a different provider.`}
```

Reply:
Being more conservative sounds ok for me, thus removing those bits.